etc/deploy/aws/aws-doco-script.sh

```diff
@@ -0,0 +1,279 @@
+#!/bin/bash
+
+# Preq: Run aws config to configure auth context profile before running this script.
+
+set -xeou pipefail
+
+jq --version || echo "This script required jq, install jq from https://stedolan.github.io/jq/download/"
```

Comment:
s/required/requires/

Also it can be installed wit' `apt` on Debian and Debian-derivatives☺